content/modules/rest/pages/entities-api/update-entities.adoc

```diff
@@ -0,0 +1,404 @@
+
+== Update Entities
+
+The Entities API also allows you to update already existing entities through the endpoint: `/entities/:entityName/:entityId` with the HTTP method `PUT`.
+
+When the entity is updated successfully, the HTTP response status code `200 - OK` is returned. By default, a JSON metadata representation of the entity is returned mainly containing the `id` attribute for further reference.
+
+[source, http request]
+.Create Customer Request
+----
+PUT http://localhost:8080/rest
+            /entities
+            /rstex11_Customer
+            /13f01f59-8e5f-4fd9-802b-66501d49ac99
+
+{
+  name: "Updated Name"
+}
+----
+
+
+[source, json]
+.Response: 200 - OK
+----
+{
+  "_entityName": "rstex11_Customer",
+  "_instanceName": "Updated Name",
+  "id": "13f01f59-8e5f-4fd9-802b-66501d49ac99"
+}
+----
+
+The Update Entity API behaves similar to the <<Create Entities,Create Entity API>> regards validation and entity referencing. The main difference is how `1:N` associations and compositions are handled. The endpoint also allows you to <<Partial Updates,partially update entities>> by only having those attributes in the request JSON.
+
+
+=== Association Attributes
+
+In case you want to update an `1:1` or `N:1` association attribute, it is enough to send the new ID of the entity to reference to. For `1:N` or `M:N` associations an updated list of entity references changes what the entity is associated to.
+
+IMPORTANT: the Update Entity endpoint will _replace_ the existing association collection for a `1:N` or `M:N` association with what is part of the request. For every entity reference that is not part of the request the reference will be removed. But, the entity that has been referenced _will not be deleted_ from the application.
```

Comment:
I would suggest using CAUTION here, as we warn the user about possible removal of data.